easybuild/easyblocks/f/ferret.py

```diff
@@ -75,15 +79,32 @@ def configure_step(self):
             (r"^INSTALL_FER_DIR =.*", "INSTALL_FER_DIR = %s" % self.installdir),
         ]
 
-        for name in deps:
-            regex_subs.append((r"^(%s.*DIR\s*)=.*" % name.upper(), r"\1 = %s" % get_software_root(name)))
+        if LooseVersion(self.version) >= LooseVersion("7.5"):
+            regex_subs.append((r"^(HDF5_LIBDIR\s*)=.*", r"\1 = %s/lib" % get_software_root('HDF5')))
+            regex_subs.append((r"^(NETCDF_LIBDIR\s*)=.*", r"\1 = %s/lib" % get_software_root('netCDF-Fortran')))
+        else:
+            for name in deps:
+                regex_subs.append((r"^(%s.*DIR\s*)=.*" % name.upper(), r"\1 = %s" % get_software_root(name)))
+
 
         if LooseVersion(self.version) >= LooseVersion("7.3"):
             regex_subs.extend([
                 (r"^DIR_PREFIX =.*", "DIR_PREFIX = %s" % self.cfg['start_dir']),
                 (r"^FER_LOCAL_EXTFCNS = $(FER_DIR)", "FER_LOCAL_EXTFCNS = $(INSTALL_FER_DIR)/libs"),
             ])
 
+        if LooseVersion(self.version) >= LooseVersion("7.5"):
+            comp_vars = {
+                'CC': 'CC',
+                'FC': 'FC',
+            }
+
+            for x, y in comp_vars.items():
```

Comment:
@smoors Please avoid using single-letter variable names outside of list comprehension

Reply:
ok, done. there were a lot of other cases like this already present.